pino.js

```diff
@@ -23,6 +23,9 @@ var defaultOptions = {
   level: 'info',
   levelVal: undefined,
   prettyPrint: false,
+  onTerminated: function () {
+    process.exit(0)
```

Comment:
we should check if we have an error, and in case exit with `process.exit(1)`, that what node would do, and what is expected unix-wise.